docs/ruleset_functions/get_opaque_surface_type.md

```diff
@@ -29,3 +29,7 @@ Logic:
   - Else determine if surface is above grade wall: ```else: surface_type = "ABOVE-GRADE WALL"```  
 
 **Returns** ```return surface_type```  
+
+**Notes:**
+
+  1. Baseline slab that is more than 24" below grade and is unregulated. Verify in RDS.
```

Comment:
Small typo - should say "Baseline slab that is more than 24" below grade is unregulated. Verify in RDS." (No "and".)